orchestrators/ProcurementOperations/tests/Unit/Coordinators/GoodsReceiptCoordinatorTest.php

```
@@ -0,0 +1,96 @@
+                    string $goodsReceiptId,
+                    string $purchaseOrderId,
+                    array $lineItems,
+                    string $receivedBy
+                ): string {
+                    return 'accrual-1';
+                }
+            },
+            purchaseOrderQuery: new class () {
+                public ?array $lastFindArgs = null;
+
+                public function findById(string $tenantId, string $id): ?object
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)